README.md

```diff
@@ -57,6 +57,46 @@ Feel free to poke around, open a new window and list or echo topics and services
 
 [![](media/localize.png)](media/localize.mp4)
 
+## Running the reconnaissance demo:
+
+Reconnaissance is the act of gathering preliminary data or intelligence on your target. The data is gathered in order to better plan for your attack. Reconnaissance can be performed actively (meaning that you are directly touching/connecting-to the target) or passively (meaning that your reconnaissance is being performed through an intermediary).
+
+The purpose of reconnaissance is to accumulate as much information as possible about a robot or robot component, including the available ROS abstractions (topics, services, etc.), the version of ROS, the target’s hardware platform and more.
+
+In this short tutorial we'll demonstrate the use of [`aztarna`](https://github.com/aliasrobotics/aztarna/), a tool for performing reconnaissance in a variety of robotic systems. Particularly, we'll look at the information we can obtain by performing active reconnaissance in an unsecure robot acting both as an attacker with direct access to the robot ("host/container" insider) and as an attacker with access to the local internal network where ROS 2 operates.
+
+### Host/container insider attacker
+``` bash
+rocker rosswg/turtlebot3_demo:roscon19 "byobu -f configs/unsecure.conf attach"
```

Comment:
This fails for me because --x11 is not passed as an argument. Adding --nvidia would be good for consistency with the rest of the tutorial

Reply:
Sent as well. Thanks!